etc/visual-studio/libopenthread-windows.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
So I fixed these, the diff now shows they are the same as before.

Interestingly, some of the files have a BOM - some do not (mostly they do)
Most do not have a final LF, some do (mostly they do not)

Not sure which editor did that to me... Ugh.